darshan-util/pydarshan/tests/test_report.py

```diff
@@ -50,7 +55,7 @@ def test_modules():
     # check if number of modules matches
     assert 4 == len(report.modules)
     assert 154 == report.modules['MPI-IO']['len']
-
+    assert False == report.modules['MPI-IO']['partial_flag']
```

Comment:
Slightly more idiomatic, though perhaps not worth changing at this point, might be:
 `assert not report.modules['MPI-IO']['partial_flag']`

Reply:
If it were me I'd probably split this off into a separate test that is parametrized over log files that both do and don't have the flag. Maybe someone can deal with that in a follow-up.